stream_srtp.go

```diff
@@ -115,15 +115,9 @@ type WriteStreamSRTP struct {
 	session *SessionSRTP
 }
 
-// WriteRTP encrypts a RTP header and its payload to the nextConn
-func (w *WriteStreamSRTP) WriteRTP(header *rtp.Header, payload []byte) (int, error) {
```

Comment:
```
         .          .    119:func (w *WriteStreamSRTP) WriteRTP(header *rtp.Header, payload []byte) (int, error) {
         .      200ms    120:	headerRaw, err := header.Marshal()
         .          .    121:	if err != nil {
         .          .    122:		return 0, err
         .          .    123:	}
         .          .    124:
         .          .    125:	// TODO(@lcurley) This will cause one, potentially two, extra allocations.
         .      230ms    126:	raw := append(headerRaw, payload...)
      40ms      9.40s    127:	return w.session.write(raw)
         .          .    128:}
```